modules/framework/galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml

```diff
@@ -2934,6 +3010,12 @@ components:
         result:
           type: string
           enum: [nil,"cancelled"]
+    GroupCancelRunStatusRequest:
```

Comment:
You haven't referenced this anywhere, should this be on line 2167?

Also, this should have a `status` section I believe like the one above? If we are just concerned with cancelling runs at the moment, then I believe the only valid result would be "cancelled" and status would be "finished".